src/processor/mapper/map_extend.cpp

```diff
@@ -59,7 +59,8 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapLogicalExtendToPhysical(
     auto boundNode = extend->getBoundNode();
     auto nbrNode = extend->getNbrNode();
     auto rel = extend->getRel();
-    auto direction = extend->getDirection();
+    auto extendDirection = extend->getDirection();
```

Comment:
Let's add a static function in this cpp file.

Maybe `getRelDataDirection(ExtendDirection extendDirection)`